storage/innobase/handler/ha_innodb.cc

```diff
@@ -881,6 +881,10 @@ static MYSQL_THDVAR_BOOL(table_locks, PLUGIN_VAR_OPCMDARG,
   /* check_func */ NULL, /* update_func */ NULL,
   /* default */ TRUE);
 
+static MYSQL_THDVAR_BOOL(strict_isolation, PLUGIN_VAR_OPCMDARG,
+  "Use strict transaction isolation.",
+  NULL, NULL, FALSE);
+
```

Comment:
I think that `innodb_snapshot_isolation` is a more descriptive name, because the aim is that with this fix (enabling write-write conflict resolution),
```sql
SET TRANSACTION ISOLATION LEVEL REPEATABLE READ;
```
behaves like its Postgres counterpart, actually implementing Snapshot Isolation.

Reply:
7899077bcbf8b4b09b5483dd0bbefca8daa5cc92 implements this.